### DISCUSSION TITLE
By default, use the caption as the alt for images in HTML

### DISCUSSION THREAD

**Discussion by yuxi-liu-wired** [quarto-dev/quarto-cli#8223] [Feature Requests]
### Description

It is necessary to have alts for images. When an image has a caption, such as in `![caption](image.png)`, the caption should be the default alt. For example, the following generated HTML has a `figcaption` but no `alt`:

```html
<div class="quarto-figure quarto-figure-center page-columns page-full">
<figure class="figure page-columns page-full">
<p><img src="figure/square_with_4_blocks.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption margin-caption">a square with 4 blocks</figcaption>
</figure>
</div>
```

## Why?

[Images must have alternate text | Axe Rules | Deque University | Deque Systems](https://dequeuniversity.com/rules/axe/4.8/image-alt?application=axeAPI)

It is good for accessibility and SEO.

Also, if the caption does not play well as an alt text, then it is probably not a good caption and needs to be rewritten for clarity.

**Comment by cderv** [comment]
@jooyoungseo do you some thoughts about this proposal ? Are caption usually good choice for default alt text ? 

I remember this was a good change in **knitr** at the time, right ? 
Discussion at : 
* https://github.com/yihui/knitr/issues/2287

Thanks.

**Reply by mcanouil** [reply]
In my opinion (FWIW), a caption is not a substitute to alternative text.
I am not sure this goes in the accessibility direction rather than in the "lazy" direction which is not in the benefits of the ones who need it.

I'd rather have accessibility warning on content rather than bad default alternative text that is unlikely to be improved at a later point.
